src/app/ui/layout/header.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from "next/link";
+import { Grid, Box } from "@radix-ui/themes";
+
+const Header: React.FC = () => {
+  return (
+    <Box as="div" className="bg-orange-200 text-black p-4 shadow">
+      <div className="mx-auto max-w-7xl grid grid-cols-[1fr,auto]">
```

Comment:
similar here, if we apply the `grid` class, we could maybe use radix-ui's `Grid`?